cmd/multiFaAcceleration/testdata/test.accel.searchspace.expected.bed

```diff
@@ -0,0 +1,151 @@
+chr1	175	225	3.810847e+00
```

Comment:
Where do these "true" values come from?  They are not pretty, so I am not thinking they were human-calcualted.  Are they from another program we "trust"?